cuqi/distribution/_gaussian.py

```diff
@@ -673,6 +673,13 @@ def get_sqrtprec_from_sqrtprec(dim, sqrtprec, sparse_flag):
         logdet = np.sum(-np.log(precision))
         rank = dim
 
+    # sqrtprec is LinearOperator (only for RTOs)
+    elif isinstance(sqrtprec, spa.linalg.LinearOperator):
```

Comment:
I think this is a very reasonable addition. I would suggest adding to the existing unit tests cases for LinearOperator that check the logpdf values.

Reply:
Done